index.js

```diff
@@ -9,6 +10,25 @@ function replaceAll(string, find, replace) {
   return string.replace(new RegExp(escapeRegExp(find), 'g'), replace)
 }
 
+function addColorSchemeMedia(isDark, color, postcss, declaration) {
+  let mediaQuery = postcss.atRule({
+    name: 'media',
+    params: `(prefers-color-scheme: ${isDark ? 'dark' : 'light'})`
```

Comment:
Space will not work if you apply it after minification.

Do we have it originally? If not, let’s just remove space.